src/building/quickstart.md

```diff
@@ -0,0 +1,60 @@
+# Quickstart
+
+This is a quickstart guide about getting the compiler running. For more information in the individual steps,
+see the other pages in this chapter.
+
+First, clone the repository:
+
+```sh
+git clone https://github.com/rust-lang/rust.git
+cd rust
+```
+
+When building the compiler, we don't use `cargo` directly, instead we use a wrapper called "x".
+On Unix-like platforms, use `./x`. On Windows, use `.\x.ps1`. This guide will use `./x`.
```

Comment:
`./x` *should* work on Windows too, if run from powershell, because it'll autocomplete the extension.

Reply:
oh, i will remove the sentence about windows and hope no one gets confused